### ISSUE TITLE
Example project

### ISSUE THREAD

**Issue by cosm1c** [stejskal/sbt-webpack#4]
Could you please provide an example project showing usage of the plugin?

**Comment by stejskal** [comment]
Good idea.  Ill set some time aside soon to create a little example project.

**Comment by gmsa** [comment]
Would also love to see an example project!

**Comment by benmccann** [comment]
It would be good to make a scripted test to help test the plugin. It could serve as an example project
